### PR TITLE
Bluetooth: Mesh: Light Ctrl sample doc fixes

### PR DESCRIPTION
**Pull request by trond-snekvik**
Fixes several inconsistencies in the Light Controller sample
documentation.

Signed-off-by: Trond Einar Snekvik <Trond.Einar.Snekvik@nordicsemi.no>

### CODE REVIEW

File: samples/bluetooth/mesh/light_ctrl/README.rst [hunk 1]
```diff
@@ -100,7 +102,7 @@ LEDs:
 Building and running
 ********************
 
-.. |sample path| replace:: :file:`samples/bluetooth/mesh/light_lc`
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/light_ctrl`
```
Comment (utzig) [inline samples/bluetooth/mesh/light_ctrl/README.rst:105]: Btw, the `sample.yaml` also needs to be updated with a similar change.

Comment (trond-snekvik) [reply samples/bluetooth/mesh/light_ctrl/README.rst:105]: Thanks!

**Review by joerchan — APPROVED** [review]

**Review by utzig — APPROVED** [review]

File: samples/bluetooth/mesh/light_ctrl/README.rst [hunk 2]
```diff
@@ -53,20 +53,22 @@ The following table shows the Mesh light controller composition data for this sa
...
-  In this sample the Light Control Server is bound internally to the Light Lightness Server instance in the second element, and enabled by default on boot up.
-  External control of the light source associated with the Light Lightness Server can be achieved indirectly by the Light Control Server, by using the Generic OnOff Server in the second element.
+- The seven other models in the first element are the product of a single instance of the Light Lightness Server.
+  The application implements callbacks for the Light Lightness Server to control the first LED on the device using the PWM driver.
+- The three models in the second element are the product of a single instance of the Light Control Server.
+  The Light Control Server controls the Light Lightness Server in the first element, deciding parameters such as fade time, lighting levels for different states, and inactivity timing.
+  In this sample, the Light Control Server is enabled by default on boot up.
```
Comment (mia-ko) [inline samples/bluetooth/mesh/light_ctrl/README.rst:64]:
```suggestion
  In this sample, the Light Control Server is enabled by default at startup.
```

**Review by mia-ko — CHANGES_REQUESTED** [review]

**Review by mia-ko — APPROVED** [review]

**Review by Andrewpini — COMMENTED** [review]
LGMT

**Pull request merged**